src/platform/forms/save-in-progress/SaveInProgressIntro.jsx

```diff
@@ -60,12 +60,15 @@ class SaveInProgressIntro extends React.Component {
 
         if (!isExpired) {
           const lastSavedDateTime = savedAt.format('M/D/YYYY [at] h:mm a');
-
+          const H = `h${this.props.headingLevel}`;
```

Comment:
`headingLevel` already exists as of #17282

(this is just a statement for my own sake, no response needed)